### PR TITLE
Updated newfeature

### PR DESCRIPTION
**Pull request by thefrau00**
chgange

### CODE REVIEW

File: newfeature [hunk 1]
```diff
@@ -1 +1 @@
-feature 01
+feature 01 - thefrau00
```
Comment (mortlake) [inline newfeature:1]: ok

File: newfeature [hunk 2]
```diff
@@ -1 +1 @@
-feature 01
```
Comment (mortlake) [inline newfeature:1]: not too bad

**Review by mortlake — COMMENTED** [review]
stuff

File: newfeature (hunk 2 above)
Comment (thefrau00) [reply newfeature:1]: what

**Pull request merged**